packages/jwt-verifier/test/spec/verifier.spec.js

```diff
@@ -29,7 +29,7 @@ const REDIRECT_URI = constants.REDIRECT_URI
 const OKTA_TESTING_DISABLEHTTPSCHECK = constants.OKTA_TESTING_DISABLEHTTPSCHECK
 
 // Some tests makes LIVE requests using getAccessToken(). These may take much longer than normal tests
-const LONG_TIMEOUT = 20000;
+const LONG_TIMEOUT = 60000;
```

Comment:
Ouch

Reply:
It really should never ever take that long. Yet when I had it at 20 there would be failures sometimes. With the perf testing under heavy load, I think it could take 30 seconds to do a full login flow.